### PR TITLE
RFC: allow major text changes to ship in minor releases

### PR DESCRIPTION
**Pull request by abouteiller**
Let the PR Author decide what is the target release for their PR, and
give the ASC flexibility in accepting the proposal to the next major
(rather than minor) release, without pushing back the proposal.

### CODE REVIEW

File: pmix_governance.tex [hunk 1]
```diff
@@ -1239,6 +1239,10 @@ \subsubsection{Major Text Changes}%
 the document such as introductory text in a chapter, definitions of terms,
 re-organization of chapter contents or chapter placement, non-errata level
 changes to descriptions of APIs including advice to implementors/users.
+A major text change can target an upcoming minor, or major release of the
+standard, but significant changes in the semantic of existing
```
Comment (jjhursey) [inline pmix_governance.tex:1243]:
```suggestion
standard, but significant changes in the semantics of existing
```

**Review by jjhursey — COMMENTED** [review]
One minor typo. Generally I think this is fine. It gives the ASC a bit more power to include descriptive items in minor releases at their discretion. Since there is already a 2 vote sequence to get things in there is plenty of time for folks to review and consider the semantic implications.

**Comment by abouteiller** [comment]
Please use emoji reactions ON THIS COMMENT to indicate your position on this proposal.

* You do not need to vote on every proposal
* If you have no opinion, don't vote - that is also useful data
* If you've already commented on this issue, please still vote so
      we know your current thoughts
* Not all proposals solve exactly the same problem, so we may end
      up accepting proposals that appear to have some overlap
This is not a binding majority-rule vote, but it will be a very
significant input into the corresponding ASC decision.

Here are the meanings for the emojis:

* Hooray or Rocket: I support this so strongly that I
      want to be an advocate for it
* Heart: I think this is an ideal solution
* Thumbs up: I'd be happy with this solution
* Confused: I'd rather we not do this, but I can tolerate it
* Thumbs down: I'd be actively unhappy, and may even consider
      other technologies instead
If you want to explain in more detail, feel free to add another
comment, but please also vote on this comment.

File: pmix_governance.tex [hunk 2]
```diff
@@ -1239,6 +1239,10 @@ \subsubsection{Major Text Changes}%
 the document such as introductory text in a chapter, definitions of terms,
 re-organization of chapter contents or chapter placement, non-errata level
 changes to descriptions of APIs including advice to implementors/users.
+A major text change can target an upcoming minor, or major release of the
```
Comment (kathrynmohror) [inline pmix_governance.tex:1242]:
```suggestion
A major text change can target an upcoming minor or major release of the
```

**Comment by jjhursey** [comment]
PMIx ASC 1Q 2022

 * Passed the first vote: 11 yes / 0 no / 0 abstain
 * This is eligible for a second vote in the 2Q 2022 ASC quarterly meeting

**Comment by jjhursey** [comment]
PMIx ASC 2Q 2022
 * Passed the second vote: 11 yes / 0 no / 0 abstain
 * Will be merged into the next release

**Review by jjhursey — APPROVED** [review]

**Comment by jjhursey** [comment]
I rebased this PR to prepare for the merge. No text changes.

**Pull request merged**